kvmagent/kvmagent/test/ha_testsuite/test_storage_fencer.py

```
@@ -0,0 +1,116 @@
+import unittest
+from kvmagent.plugins import ha_plugin
+from mock import patch
```

Ruff (0.8.2)

`mock.patch` imported but unused

Remove unused import: `mock.patch`

(F401)